confdgnmi/testtool/OpenConfig/interfaces.resource

```diff
@@ -12,12 +12,14 @@ Get container includes
     Verify Get of  ${path}
     Check last updates include   ${leaf_name}
 
-Get interface "config" includes leaf
+Iterate interface "config" includes leaf
     [Arguments]    ${leaf_name}
     DataType set to    CONFIG
     Get container includes  /interfaces/interface[name=${OC_INTERFACE}]/config  ${leaf_name}
+    [Teardown]    Teardown gNMI state
```

Comment:
I'm not terribly happy about keywords having their teardown procedure, but I assume you tried to find better solution; I don't see any.

Reply:
I agree, though this seems to be recommended approach on various discussions/forums online for this case - when `[Template]` tests behave like separate/standalone ones and need a teardown, that is NOT executed by ROBOT framework itself here.
I tried to mark the affected keywords with `Iterate` prefix to help a bit with a naming pattern.